src/makeCalculator.js

```diff
@@ -37,7 +37,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+
+    add(num) {
+      calculator.result += num;
+
+      return this;
+    },
```

Comment:
You don't need return here, I think)

Reply:
I will try to remove return, but i'm not sure if it will work properly)

Reply:
You don't call these methods explicitly, so indeed you don't need it